test/unit/docsify.test.js

```diff
@@ -0,0 +1,133 @@
+/* global before after */
+/* eslint-disable no-global-assign */
+require = require('esm')(module /* , options */);
+
+const http = require('http');
+const handler = require('serve-handler');
+const { expect } = require('chai');
+const { initJSDOM } = require('../_helper');
+
+const docsifySite = 'http://127.0.0.1:3000';
+
+const markup = /* html */ `<!DOCTYPE html>
+  <html>
+    <head></head>
+    <body>
+      <div id="app"></div>
+      <script src="/lib/docsify.js"></script>
```

Comment:
@anikethsaha This is where it loads the built output file.

Reply:
cool